content/MBC1.md

```diff
@@ -1,12 +1,12 @@
 (max 2MByte ROM and/or 32KByte RAM)
 
-This is the first MBC chip for the Game Boy. Any newer MBC chips are
-working similiar, so that is relative easy to upgrade a program from one
-MBC chip to another - or even to make it compatible to several different
+This is the first MBC chip for the Game Boy. Any newer MBC chips 
+work similarly, so it is relatively easy to upgrade a program from one
+MBC chip to another - or to make it compatible with several different
 types of MBCs.
 
-Note that the memory in range 0000-7FFF is used for both reading from
-ROM, and for writing to the MBCs Control Registers.
+Note that the memory in range 0000-7FFF is used both for reading from
+ROM, and writing to the MBCs Control Registers.
```

Comment:
No comma before the preposition in a list of two items

Reply:
Do I have to edit that in, or did you already change it?